src/Parallax.js

```diff
@@ -834,6 +849,14 @@ var LayerView = exports.LayerView = Class(View, function () {
     var s = this.style;
     return s.scale * (y - s.anchorY) + s.anchorY;
   };
+
+  this.spawnAllowedByPerformance = function () {
+    if (!this.performanceCutoff) {
```

Comment:
Can we make this part of the `validateSpawn` function, and then remove the additional checks in each piece spawn function? I think that's the only substantive thing I would change. I was also wondering if `performanceCutoff` could ever be 0 and have a valid meaning, but I don't think it would.